<?xml version="1.0" encoding="UTF-8"?>
<testsuite name="CardGameTest" tests="1" skipped="0" failures="1" errors="0" timestamp="2023-07-24T18:19:39" hostname="DESKTOP-I1PJDPK" time="0.363">
  <properties/>
  <testcase name="start" classname="CardGameTest" time="0.363">
    <failure message="java.lang.AssertionError: Exception in test #2&#10;&#10;java.lang.NumberFormatException: For input string: &quot;one&quot;&#10;&#9;at java.base/java.lang.NumberFormatException.forInputString(NumberFormatException.java:67)&#10;&#9;at java.base/java.lang.Integer.parseInt(Integer.java:668)&#10;&#9;at java.base/java.lang.Integer.parseInt(Integer.java:786)&#10;&#9;at indigo.Deck.get(Main.kt:58)&#10;&#9;at indigo.MainKt.main(Main.kt:89)&#10;&#9;at indigo.MainKt.main(Main.kt)&#10;&#10;Please find below the output of your program during this failed test.&#10;Note that the '&gt;' character indicates the beginning of the input line.&#10;&#10;---&#10;&#10;Choose an action (reset, shuffle, get, exit):&#10;&gt; get&#10;Number of cards:&#10;&gt; 0&#10;Invalid number of cards.&#10;Choose an action (reset, shuffle, get, exit):&#10;&gt; get&#10;Number of cards:&#10;&gt; one" type="java.lang.AssertionError">java.lang.AssertionError: Exception in test #2

java.lang.NumberFormatException: For input string: &quot;one&quot;
	at java.base/java.lang.NumberFormatException.forInputString(NumberFormatException.java:67)
	at java.base/java.lang.Integer.parseInt(Integer.java:668)
	at java.base/java.lang.Integer.parseInt(Integer.java:786)
	at indigo.Deck.get(Main.kt:58)
	at indigo.MainKt.main(Main.kt:89)
	at indigo.MainKt.main(Main.kt)

Please find below the output of your program during this failed test.
Note that the '&gt;' character indicates the beginning of the input line.

---

Choose an action (reset, shuffle, get, exit):
&gt; get
Number of cards:
&gt; 0
Invalid number of cards.
Choose an action (reset, shuffle, get, exit):
&gt; get
Number of cards:
&gt; one
	at org.junit.Assert.fail(Assert.java:89)
	at org.hyperskill.hstest.stage.StageTest.start(StageTest.java:203)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:77)
	at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.base/java.lang.reflect.Method.invoke(Method.java:568)
	at org.junit.runners.model.FrameworkMethod$1.runReflectiveCall(FrameworkMethod.java:59)
	at org.junit.internal.runners.model.ReflectiveCallable.run(ReflectiveCallable.java:12)
	at org.junit.runners.model.FrameworkMethod.invokeExplosively(FrameworkMethod.java:56)
	at org.junit.internal.runners.statements.InvokeMethod.evaluate(InvokeMethod.java:17)
	at org.junit.runners.ParentRunner$3.evaluate(ParentRunner.java:306)
	at org.junit.runners.BlockJUnit4ClassRunner$1.evaluate(BlockJUnit4ClassRunner.java:100)
	at org.junit.runners.ParentRunner.runLeaf(ParentRunner.java:366)
	at org.junit.runners.BlockJUnit4ClassRunner.runChild(BlockJUnit4ClassRunner.java:103)
	at org.junit.runners.BlockJUnit4ClassRunner.runChild(BlockJUnit4ClassRunner.java:63)
	at org.junit.runners.ParentRunner$4.run(ParentRunner.java:331)
	at org.junit.runners.ParentRunner$1.schedule(ParentRunner.java:79)
	at org.junit.runners.ParentRunner.runChildren(ParentRunner.java:329)
	at org.junit.runners.ParentRunner.access$100(ParentRunner.java:66)
	at org.junit.runners.ParentRunner$2.evaluate(ParentRunner.java:293)
	at org.junit.runners.ParentRunner$3.evaluate(ParentRunner.java:306)
	at org.junit.runners.ParentRunner.run(ParentRunner.java:413)
	at org.gradle.api.internal.tasks.testing.junit.JUnitTestClassExecutor.runTestClass(JUnitTestClassExecutor.java:108)
	at org.gradle.api.internal.tasks.testing.junit.JUnitTestClassExecutor.execute(JUnitTestClassExecutor.java:57)
	at org.gradle.api.internal.tasks.testing.junit.JUnitTestClassExecutor.execute(JUnitTestClassExecutor.java:39)
	at org.gradle.api.internal.tasks.testing.junit.AbstractJUnitTestClassProcessor.processTestClass(AbstractJUnitTestClassProcessor.java:62)
	at org.gradle.api.internal.tasks.testing.SuiteTestClassProcessor.processTestClass(SuiteTestClassProcessor.java:52)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:77)
	at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.base/java.lang.reflect.Method.invoke(Method.java:568)
	at org.gradle.internal.dispatch.ReflectionDispatch.dispatch(ReflectionDispatch.java:36)
	at org.gradle.internal.dispatch.ReflectionDispatch.dispatch(ReflectionDispatch.java:24)
	at org.gradle.internal.dispatch.ContextClassLoaderDispatch.dispatch(ContextClassLoaderDispatch.java:33)
	at org.gradle.internal.dispatch.ProxyDispatchAdapter$DispatchingInvocationHandler.invoke(ProxyDispatchAdapter.java:94)
	at jdk.proxy1/jdk.proxy1.$Proxy2.processTestClass(Unknown Source)
	at org.gradle.api.internal.tasks.testing.worker.TestWorker$2.run(TestWorker.java:176)
	at org.gradle.api.internal.tasks.testing.worker.TestWorker.executeAndMaintainThreadName(TestWorker.java:129)
	at org.gradle.api.internal.tasks.testing.worker.TestWorker.execute(TestWorker.java:100)
	at org.gradle.api.internal.tasks.testing.worker.TestWorker.execute(TestWorker.java:60)
	at org.gradle.process.internal.worker.child.ActionExecutionWorker.execute(ActionExecutionWorker.java:56)
	at org.gradle.process.internal.worker.child.SystemApplicationClassLoaderWorker.call(SystemApplicationClassLoaderWorker.java:113)
	at org.gradle.process.internal.worker.child.SystemApplicationClassLoaderWorker.call(SystemApplicationClassLoaderWorker.java:65)
	at worker.org.gradle.process.internal.worker.GradleWorkerMain.run(GradleWorkerMain.java:69)
	at worker.org.gradle.process.internal.worker.GradleWorkerMain.main(GradleWorkerMain.java:74)
</failure>
  </testcase>
  <system-out><![CDATA[?[1;31m
Start test 1?[0m
Choose an action (reset, shuffle, get, exit):
> reset
Card deck is reset.
Choose an action (reset, shuffle, get, exit):
> get
Number of cards:
> 10
A♦ A♥ A♠ A♣ 2♦ 2♥ 2♠ 2♣ 3♦ 3♥
Choose an action (reset, shuffle, get, exit):
> get
Number of cards:
> 43
The remaining cards are insufficient to meet the request.
Choose an action (reset, shuffle, get, exit):
> reset
Card deck is reset.
Choose an action (reset, shuffle, get, exit):
> get
Number of cards:
> 52
A♦ A♥ A♠ A♣ 2♦ 2♥ 2♠ 2♣ 3♦ 3♥ 3♠ 3♣ 4♦ 4♥ 4♠ 4♣ 5♦ 5♥ 5♠ 5♣ 6♦ 6♥ 6♠ 6♣ 7♦ 7♥ 7♠ 7♣ 8♦ 8♥ 8♠ 8♣ 9♦ 9♥ 9♠ 9♣ 10♦ 10♥ 10♠ 10♣ J♦ J♥ J♠ J♣ Q♦ Q♥ Q♠ Q♣ K♦ K♥ K♠ K♣
Choose an action (reset, shuffle, get, exit):
> get
Number of cards:
> 1
The remaining cards are insufficient to meet the request.
Choose an action (reset, shuffle, get, exit):
> exit
Bye
?[1;31m
Start test 2?[0m
Choose an action (reset, shuffle, get, exit):
> get
Number of cards:
> 0
Invalid number of cards.
Choose an action (reset, shuffle, get, exit):
> get
Number of cards:
> one
]]></system-out>
  <system-err><![CDATA[]]></system-err>
</testsuite>
